x-pack/legacy/plugins/integrations_manager/common/types.ts

```diff
@@ -10,14 +10,15 @@ export { Request, ResponseToolkit, Server, ServerRoute } from 'hapi';
 
 export type InstallationStatus = Installed['status'] | NotInstalled['status'];
 
-export type AssetType =
-  | 'config'
-  | 'dashboard'
-  | 'index-pattern'
-  | 'ingest-pipeline'
-  | 'search'
-  | 'timelion-sheet'
-  | 'visualization';
+export enum AssetType {
+  config = 'config',
+  dashboard = 'dashboard',
+  visualization = 'visualization',
+  search = 'search',
+  ingestPipeline = 'ingest-pipeline',
+  indexPattern = 'index-pattern',
+  timelionSheet = 'timelion-sheet',
```

Comment:
Not for this PR but more general thought on this: The reason I prefixed the asset always by the service is to prevent conflicts in the future and make them unique. So if there is an `index-pattern` in Kibana and Elasticsearch, both is possible. Because of this I think we should start to refer to either `elasticsearch/ingest-pipeline` or have an asset type as a combination of the service and type.

Reply:
Good point. At the moment some of these double as Kibana saved-object types (as defined here: https://www.elastic.co/guide/en/kibana/master/saved-objects-api-get.html) so we'll probably want to uncouple our asset types from those and only map between them.